sdk/keyvault/azcertificates/utils_test.go

```diff
@@ -58,12 +58,6 @@ func TestMain(m *testing.M) {
 		if err != nil {
 			panic(err)
 		}
-
```

Comment:
Don't you still need this? I guess we'll always use a particular tenant, but if there were an external contributor or even one of us testing against a personal account we'd want this to be replaced, right?

Reply:
No. We need a tenant ID in the recorded `WWW-Authenticate` header, to prevent a parsing error in the challenge authentication policy, but the value doesn't matter because it isn't used in matching and we don't authenticate in playback. Since the live tenant ID isn't secret, we can simply record that.